docs/installation/020_INSTALL_MANUAL.md

```diff
@@ -208,7 +157,7 @@ manager, please follow these steps:
     time you try to generate an image. Fortunately, after the warm-up period
     rendering will be fast.
 
-10. Subsequently, to relaunch the script, be sure to enter `InvokeAI` directory,
+9.  Subsequently, to relaunch the script, be sure to enter `InvokeAI` directory,
     activate the virtual environment, and then launch `invoke.py` script.
```

Comment:
Change this to the `invokeai`.

Reply:
So What I also found out, and is imho way more usefull:

```py
python -m venv .venv \ # Create the venv as more common .venv
  --prompt InvokeAI \  # still make the prompt show InvokeAI to be identified easy
  --upgrade-deps       # directly upgrade pip wheel and setuptools in the venv
```

so would this be better vs leaving the kind of common .venv folder name?

Reply:
it is not invokeai yet?! But Ok...